### PR TITLE
PATRON2020-16 ServoControllerHVAC

### PR DESCRIPTION
**Pull request by kuba8686**
Feature to be updated:
- implementation of simple servo controller using Servo.h library
- servo position (angle) set by serial port (testing)

### CODE REVIEW

File: ServoControllerHVAC/ServoControllerHVAC.ino [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include <Servo.h>
+
+#define PIN 15
+
+Servo servo;
+
+int servoAng;
+
+void setup() {
```
Comment (Grzesm) [inline ServoControllerHVAC/ServoControllerHVAC.ino]: Curly bracket is in the same lane with function definition.

File: ServoControllerHVAC/ServoControllerHVAC.ino [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  servo.attach(PIN);
+  servo.write(10);
+  Serial.begin(9600);
+}
+
+
+void loop() 
+{
```
Comment (Grzesm) [inline ServoControllerHVAC/ServoControllerHVAC.ino]: Here it is okay.

File: ServoControllerHVAC/ServoControllerHVAC.ino (hunk 1 above)
Comment (kuba8686) [reply ServoControllerHVAC/ServoControllerHVAC.ino]: fixed

File: ServoControllerHVAC/ServoControllerHVAC.ino [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+#define PIN 15
+
+Servo servo;
+
+int servoAng;
+
+void setup() 
+{
+  servo.attach(PIN);
```
Comment (Siwy160) [inline ServoControllerHVAC/ServoControllerHVAC.ino]: 4 spaces as indends.

Comment (kuba8686) [reply ServoControllerHVAC/ServoControllerHVAC.ino]: fixed

File: ServoControllerHVAC/ServoControllerHVAC.ino [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+#define PIN 15
+
+Servo servo;
+
+int servoAng;
+
+void setup() {
+  servo.attach(PIN);
+  servo.write(10);
+  Serial.begin(9600);
```
[the remark this answers is not in the document]
Comment (kuba8686) [reply ServoControllerHVAC/ServoControllerHVAC.ino]: It will. Serial just for testing at the moment.

**Pull request merged**